streamdeck_ui/gui.py

```diff
@@ -137,12 +139,19 @@ def handle_keypress(ui, deck_id: str, key: int, state: bool) -> None:
 
     # TODO: Handle both key down and key up events in future.
     if state:
+        page = api.get_page(deck_id)
+
+        run_when_locked = api.get_button_run_when_locked(deck_id, page, key)
+        is_system_locked = system_api.get_is_system_locked()
```

Comment:
This can avoid the dbus work _and added button press latency_ if the key is allowed to run when the system is locked.

> if not run_when_locked:
>   if system_api.get_is_system_locked():
>    print and return